src/elements/ts-main/ts-dashboard.html

```diff
@@ -306,6 +453,51 @@
           }, 500);
       },
 
+      openimport: function () {
+          //this.set('currentinfo', e.model.proj);
+
+
+          console.log("opening import");
+          console.log("project list",this.projectlist);
+          var mythis = this;
+          var chooser = this.$.import;
+          chooser.value = null;
+          chooser.addEventListener('change', function(evt) {
+              mythis.$['import-dialog'].open();
+          }, false);
+          chooser.click();
+      },
+
+      startimport: function(e){
+          var loading = this.$.loading;
+          var mythis = this;
+          var chooser = this.$.import;
+          var proj = e.model.proj;
+          this.$['import-dialog'].close();
+          console.log("selected project",proj);
+          mythis.set('options.body', "Importing.  Please wait...");
+          mythis.set('options.loading', true);
+          loading.open();
+          for(var i=0; i < chooser.files.length; i ++) {
+              var fileExt = chooser.files[i].name.split('.').pop().toLowerCase();
+              if(fileExt === "zip" || fileExt === "txt" || fileExt === "usfm"){
+                  App.importer.importUSFMFile(chooser.files[0],proj,this.currentuser).then(function(){
```

Comment:
We need the user to identify which target language the translation is in as well. This is the same list of languages they see when they start a new translation. Be sure to include a search bar in this dialog.